pkg/synopsysctl/cmd_update.go

```diff
@@ -1066,24 +941,12 @@ func init() {
 
 	updateBlackDuckCmd.AddCommand(updateBlackDuckRootKeyCmd)
 
-	updateBlackDuckAddPVCCmd.Flags().StringVar(&blackDuckPVCSize, "size", blackDuckPVCSize, "Size of the PVC")
-	updateBlackDuckAddPVCCmd.Flags().StringVar(&blackDuckPVCStorageClass, "storage-class", blackDuckPVCStorageClass, "Storage class name")
-	updateBlackDuckAddPVCCmd.Flags().StringVarP(&mockFormat, "mock", "o", mockFormat, "Prints the new CRD resource spec in the specified format instead of editing it [json|yaml]")
-	updateBlackDuckAddPVCCmd.Flags().StringVarP(&mockKubeFormat, "mock-kube", "k", mockKubeFormat, "Prints the new Kubernetes resource specs in the specified format instead of editing them [json|yaml]")
-	updateBlackDuckCmd.AddCommand(updateBlackDuckAddPVCCmd)
-
-	updateBlackDuckAddEnvironCmd.Flags().StringVarP(&mockFormat, "mock", "o", mockFormat, "Prints the new CRD resource spec in the specified format instead of editing it [json|yaml]")
```

Comment:
Is this is being deleted from addEnv, shouldn't it be deleted from AddRegistry as well?  Why are these options being deleted?